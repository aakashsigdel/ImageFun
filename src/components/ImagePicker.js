import React from 'react';

const ImagePicker = props =>
  <input type="file" onChange={console.log} />;

export default ImagePicker;

