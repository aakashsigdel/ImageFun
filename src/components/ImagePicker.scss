.image-chooser {
  margin: 20px;
  border: 1px solid lightgrey;
  padding: 10px;
}
