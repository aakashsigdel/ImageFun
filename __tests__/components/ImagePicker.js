import React from 'react';
import ImagePicker from '../../src/components/ImagePicker';

import renderer from 'react-test-renderer';

describe('<ImagePicker /> component', () => {
  it('should render <ImagePicker /> correctly', () => {
    const tree = renderer.create(<ImagePicker />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
