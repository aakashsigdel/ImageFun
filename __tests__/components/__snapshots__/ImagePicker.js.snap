exports[`<ImagePicker /> component should render <ImagePicker /> correctly 1`] = `
<input
  onChange={[Function]}
  type="file" />
`;
